docs/Actions.md

```diff
@@ -152,8 +152,8 @@ const UserProfile = ({ userId }) => {
 
     return (
         <ul>
-            <li>Name: {data.name}</li>
-            <li>Email: {data.email}</li>
+            <li>Name: {data.data.name}</li>
```

Comment:
Shouldn't we try to set this more elegantly with: 

```
const { data: result, isLoading, error } = useQuery(
...
<li>Name: {result.data.name}</li>
<li>Name: {result.data. email}</li>
``` 
```?

Reply:
Not sure it's better...